chapter3.md

```diff
@@ -64,26 +64,26 @@ Averaging the AICc weights and distinct abundance values across all sites in a d
 
 <!--Could be a terrestrial vs. marine thing, although fish showed the same pattern.  The fish data was a little over half freshwater, a few estuary, and the rest marine, but a lot of near shore, mangrove-y areas.  No deep water sites.-->
-In combination our results and those of Connolly et al. (2014) suggest that while marine systems are generally approximated by non-neutral dynamics,  terrestrial systems show more variability between neutral and non-neutral dynamics. Several studies have noted that both patterns and processes may vary between marine and terrestrial systems [@webb2012; @horne2015]. However, while macroecological patterns have not been studied as extensively in marine systems as in terrestrial systems, marine and terrestrial systems do tend to exhibit many of the same general macroecological patterns [@webb2012]. However, Webb (2012) points out that while the same general patterns may occur, the processes generating those patterns may be different [@webb2012]. This could lead to differences in the 'second order effects' noted by Blonder et al. (2014) as the most promising avenue for identifying process using macroecological patterns like the SAD.
+In combination, our results and those of Connolly et al. (2014) suggest that while marine systems are generally approximated by non-neutral dynamics,  terrestrial systems show more variability between neutral and non-neutral dynamics. Several studies have noted that both patterns and processes may vary between marine and terrestrial systems [@webb2012; @horne2015]. While macroecological patterns have not been studied as extensively in marine systems as in terrestrial systems, marine and terrestrial systems do tend to exhibit many of the same general macroecological patterns [@webb2012]. However, Webb (2012) points out that while the same general patterns may occur, the processes generating those patterns may be different [@webb2012]. This could lead to subtle differences in the details of the patterns ('second order effects'), noted by Blonder et al. (2014) as the most promising avenue for identifying process using macroecological patterns like the SAD.
```

Comment:
Much better. One more tweak:

"('second order effects'), noted by Blonder et al. (2014)"
->
"('second order effects'), which Blonder et al. (2014) noted"

and

"macroecological patterns like the SAD." -> "macroecological patterns."